content/docs/command-reference/plots/index.md

```diff
@@ -98,72 +100,50 @@ names in the `train` array below:
+- `linear` - linear plot, used by default
 - `scatter` - scatter plot
 - `smooth` - linear plot with LOESS smoothing, see
   [example](/doc/command-reference/plots#example-smooth-plot)
 - `confusion` - confusion matrix, see
   [example](/doc/command-reference/plots#example-confusion-matrix)
+- `confusion_normalized` - confusion matrix with values normalized to <0, 1>
+  range
+- `simple` - simplest linear template, good for learning how to work with
+  [`templates`](/doc/command-reference/plots/templates) command
 
-### Custom templates
-
-Plot template files are [Vega-Lite](https://vega.github.io/vega-lite/) files
-that use predefined DVC anchors as placeholders for DVC to inject the plot
-values. You can create a custom template from scratch, or modify an existing one
-from `.dvc/plots/`.
-
-💡 Note that custom templates can be safely added to the template directory.
+File targets given to `dvc plots show` and `dvc plots diff` are treated as
+separate data series, each to be injected into a template file. There are two
+important fields that DVC adds to the plot data:
 
-All metrics files given to `dvc plots show` and `dvc plots diff` as input are
-combined together into a single data array for injection into a template file.
-There are two important fields that DVC adds to the plot data:
+- `step` - zero-based counter for the data rows/values. In many cases it
+  corresponds to a machine learning training epoch number.
 
-- `index` - zero-based counter for the data rows/values. In many cases it
-  corresponds to a machine learning training epoch or step number.
-
-- `rev` - Git commit hash, tag, or branch of the metrics file. This helps
-  distinguish between different versions when using the `dvc plots diff`
-  command.
+- `rev` - This field helps distinguish between data sourced from different
+  revisions, files or columns.
 
 Note that in the case of CSV/TSV metrics files, column names from the table
 header (first row) are equivalent to field names.
 
-### DVC template anchors
-
-- `<DVC_METRIC_DATA>` (**required**) - the plot data from any type of metrics
-  files is converted to a single JSON array, and injected instead of this
-  anchor. Two additional fields will be added: `index` and `rev` (explained
-  above).
-
-- `<DVC_METRIC_TITLE>` (optional) - a title for the plot, that can be defined
-  with the `--title` option of the `dvc plot` subcommands.
-
-- `<DVC_METRIC_X>` (optional) - field name of the data for the X axis. It can be
-  defined with the `-x` option of the `dvc plot` subcommands. The auto-generated
-  `index` field (explained above) is the default.
-
-- `<DVC_METRIC_Y>` (optional) - field name of the data for the Y axis. It can be
-  defined with the `-y` option of the `dvc plot` subcommands. It defaults to the
-  last header of the metrics file: the last column for CSV/TSV, or the last
-  field for JSON/YAML.
-
-- `<DVC_METRIC_X_LABEL>` (optional) - field name to display as the X axis label
+### Custom templates
 
```

Comment:
Prob not needed if we link to `plot templates` per one of my suggestions above.
```suggestion
```

Reply:
Not sure how to proceed, didn't remove it because I followed @dberenbaum suggestion from 
https://github.com/iterative/dvc.org/pull/3697#discussion_r908790245

Reply:
Np let's leave it for now. Thanks